src/components/Menu/Menu.stories.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+
+import Menu from './Menu';
+import MenuTopContent from '../MenuTopContent';
+import { menu, argTypes } from './Menu.stories.tsx';
+
+export const MenuTopContentTemplate = (args) => {
+  return <MenuTopContent {...args} />;
+};
+
+<Meta title='Molecules/Menu' component={Menu} argTypes={argTypes} />
+
+# Menu
+
+A `Menu` displays a list of choices on a temporary surfaces, or popovers. It appears when a user interacts with a button or some other control.
+`Menus` are meant to be less disruptive to a user's current context compared to `Dialogs`.
+
+A `Menu` can be composed of `MenuTopContent` and any number of `MenuItems` or any number of `Buttons`. Choosing an option like clicking a button
+should immediately commit the the option and close the menu.
+
+The most notable props that `Menu` accepts are `anchorEl` which sets the position of the menu to the referenced anchor. The `open` prop determines if
+the menu is open or not. `MenuItems`, `MenuTopContent`, and `Buttons` are passed as children.
+
+# MenuTopContent
+
+`MenuTopContent` is a custom component that displays a title for the menu. It takes an optional `title` prop and an optional `onClose` prop.
+If `onClose` is defined, `MenuTopContent` will display an `X` (close) icon.
+
+<Canvas>
+  <Story story={menu} name='Playground' />
+</Canvas>
+
+<ArgsTable of={Menu} story='Playground' />
```

Comment:
nit: I'm moving towards calling this area "Sandbox" (though I'm not _that_ strongly opinionated about it)